package.json

```diff
@@ -62,11 +62,14 @@
 		"@babel/preset-typescript": "^7.15.0",
 		"@sentry/node": "^7.7.0",
 		"@sentry/tracing": "^7.7.0",
+		"@yukikaze-bot/erlpack": "^1.0.1",
 		"ajv": "^8.11.0",
 		"ajv-formats": "^2.1.1",
 		"amqplib": "^0.10.1",
+		"bcrypt": "^5.0.1",
```

Comment:
Some environments don't support this package, this is why is not included by default.

Same for `canvas` package.

Reply:
package.json should not be updated to include these dependencies (might add to githook)